etm/operator-interface-spec/etm-oper-api.yaml

```diff
@@ -919,18 +1345,37 @@ components:
 
         Modifications Include:
         - Make time_start and time_end required
+        - Add 
       required:
         - volume
         - time_start
         - time_end
       type: object
       properties:
         volume:
-          $ref: "https://raw.githubusercontent.com/astm-utm/Protocol/implementation_2020q2/utm.yaml#/components/schemas/Volume3D"
+          $ref: '#/components/schemas/Volume3D'
         time_start:
-          $ref: "https://raw.githubusercontent.com/astm-utm/Protocol/implementation_2020q2/utm.yaml#/components/schemas/Time"
+          $ref: '#/components/schemas/Time'
         time_end:
-          $ref: "https://raw.githubusercontent.com/astm-utm/Protocol/implementation_2020q2/utm.yaml#/components/schemas/Time"
+          $ref: '#/components/schemas/Time'
+        containment_confidence:
+          description: Confidence of aircraft staying inside of 4D volume
+          type: number
+          format: float
+          minimum: 0
+          maximum: 1
+        confidence_distribution:
+          description: Containment confidence probability distribution type
+          type: string
+          enum:
+          - Uniform
+          - Normal
+        wind_source:
```

Comment:
hmmm this is wrong the way I read  Jaewoo's spec.  The wind source is associated with a list of volumes, not each volume. So it would be moved up to be a peer with Operation .gufi.